code/datums/diseases/pierrot_throat.dm

```diff
@@ -26,3 +26,39 @@
 		if(4)
 			if(prob(5))
 				affected_mob.say( pick( list("HONK!", "Honk!", "Honk.", "Honk?", "Honk!!", "Honk?!", "Honk...") ) )
+
+
+/datum/disease/pierrot_throat/advanced
+	name = "Advanced Pierrot's Throat"
+	max_stages = 4
```

Comment:
Most of these vars are already defined under `/datum/disease/pierrot_throat` and as such don't need to be redefined here

Reply:
I would rather keep it just incase the values need to be tweaked.

Reply:
In that case you can just redefine them if you need to - no need to have the redundancy